dna/zomes/projects/src/lib.rs

```diff
@@ -63,6 +63,48 @@ pub enum SignalType {
   GoalVote(GoalVoteSignal),
   Member(MemberSignal),
   ProjectMeta(ProjectMetaSignal),
+  EditingGoal(EditingGoalSignal),
+}
+
+#[derive(Debug, Serialize, Deserialize, SerializedBytes)]
+pub enum GoalField {
+  Title,
+  Description,
+}
+
+#[derive(Debug, Serialize, Deserialize, SerializedBytes)]
+pub struct EditingGoalSignal {
+  pub goal_field: GoalField,
+  pub goal_address: HeaderHash,
+  pub editing_agent: AgentPubKey,
+  pub is_editing: bool,
+}
+
+#[derive(Debug, Serialize, Deserialize, SerializedBytes)]
+pub struct EditingGoalInfo {
+  pub goal_field: GoalField,
+  pub goal_address: HeaderHash,
+  pub is_editing: bool,
+}
+fn convert_to_receiver_signal(signal: EditingGoalSignal) -> SignalType {
+  SignalType::EditingGoal(signal)
+}
```

Comment:
we can just omit this function and construct it inline... this convention of this function is to go with hdk_crud expectations
Just do this in the function below
SignalType::EditingGoal(signal)